birdhouse/scripts/sync-data

```diff
@@ -28,7 +28,7 @@ SOURCE_HOST="$1"; shift
 FORCE_MODE="$1"
 
 if [ -z "$SOURCE_HOST" ]; then
-    echo "ERROR: no source host provided" 1>&2
+    echo "${MSG_ERROR}no source host provided" 1>&2
```

Comment:
What's the difference between `echo "${MSG_ERROR} ...."` and `log ERROR ....` ?

Can they be both used interchangeably?  

If yes, I prefer `${MSG_ERROR}` because if it is not defined because of sourcing error, at least the error message will still be printed.  With `log ERROR`, if sourcing `read-configs.include.sh` fails, the error message won't be printed at all.

Reply:
Something I missed. It was the first iteration before `log` was introduced. Already fixed.

Reply:
Then you might want to review the entire PR.  I see this old style `MSG_` many times.  That's why I thought they can be used interchangeably.